src/components/CreateForm/CreateForm.js

```diff
@@ -81,12 +240,11 @@ const CreateForm = ({ images, setPostImage }) => {
           value={description}
           onInput={(e) => setDescription(e.target.value)}
         />
-        <Button type='submit'>POST ART</Button>
-        {/* <p>
-          {currentLocation
-            ? `${currentLocation.latitude}, ${currentLocation.longitude}`
-            : 'no location'}
-        </p> */}
+        <section className='form-btn-wrapper post-art-btn'>
+          <Button type='submit'>
+            POST ART <FaTelegramPlane />
+          </Button>
+        </section>
```

Comment:
Great job on the form. It looks super close to its final form. Will work on the async issues for displaying.